index.js

```diff
@@ -161,7 +161,7 @@ class StackUtils {
     setFile(res, site.getFileName(), this._cwd);
 
     if (site.isConstructor()) {
-      res.constructor = true;
+      Object.defineProperty(res, 'constructor', { value: true });
```

Comment:
```suggestion
      Object.defineProperty(res, 'constructor', { configurable: true, value: true });
```
just in case it doesn't already exist?

Reply:
Done